sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md

```diff
@@ -8,8 +8,19 @@
 
 ### Bugs Fixed
 
+* Will no longer emit `db.statement_type` as a part of SQL custom dimensions.
+  This attribute was removed from the SqlClient Instrumentation Library because it's not a part of the [semantic conventions](https://github.com/open-telemetry/semantic-conventions/blob/v1.24.0/docs/database/database-spans.md#call-level-attributes).
```

Comment:
Do you want to add PR link at the end?

Reply:
I'm not clean on how we want to handle this for vendored code. We increase our maintenance load if we want list every major change. 

I think a reasonable user could navigate to the otel repo to review changes there.
We can always adapt our strategy here based on feedback.

Reply:
I'm merging now so I can continue working on following changes.
We can review this again before releasing.